tasks/package.js

```diff
@@ -71,4 +146,15 @@ module.exports = function(grunt) {
     task: 'package',
     group: 'Operations'
   });
+
+  function isValidTarget(target) {
+    if (target.length) {
+      var validTargets = ['acquia'];
+      return (validTargets.indexOf(target) > -1);
+    }
+    else {
+      return false;
+    }
```

Comment:
I don't think we need to whitelist targets. Let people configure whichever keys they want to build out a target/profile. We should document any "magic keys" such as `acquia` that provide special sauce.